src/main/java/Todo.java

```diff
@@ -0,0 +1,24 @@
+public class Todo {
+    private String description;
+    private boolean isDone;
+
+    public Todo(String description) {
+        this.isDone = false;
+        this.description = description;
+    }
+    private String isDone() {
+        return this.isDone ? "[✓] " : "[✗] ";
+    }
+
+    public void setDone() {
+        this.isDone = true;
+        System.out.println("  Nice! I've marked this task as done:");
+        System.out.println("  " + "[✓] " + this.description);
+    }
+    public char getTaskType() {
+        return 'T';
```

Comment:
Consider returning a character set to 'T' instead of just putting it in the return statement